Selenium_Advanced/EpumTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+
+namespace Selenium_Advanced
+{
+    public class EpumTests
+    {
+        private IWebDriver _chrome;
+
+        private const string MainPage = "https://www.epam.com/";
+
+        private WebDriverWait Waiter { get; set; }
+
+        private TimeSpan _timeout = TimeSpan.FromSeconds(5);
+        private IJavaScriptExecutor? Jse { get; set; }
+
+        [SetUp]
+        public void Setup()
+        {
+            _chrome = new ChromeDriver();
+            _chrome.Manage().Window.Maximize();
+            Waiter = new WebDriverWait(_chrome, TimeSpan.FromSeconds(15));
+            _chrome.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            _chrome.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            _chrome.Navigate().GoToUrl(MainPage);
+            Jse = _chrome as IJavaScriptExecutor;
+        }
+
+        [SetUp]
+        public void SetupWaiter()
+        {
+            Waiter = new WebDriverWait(_chrome, _timeout);
+            Waiter.IgnoreExceptionTypes(typeof(NoSuchElementException), typeof(ElementNotVisibleException));
+
+        }
+
+        [Test]
+        public void CheckEpamJobListingsIsOpen()
+        {
+            var action = new Actions(_chrome);
+            var careersElement = _chrome.FindElement(By.XPath("//*[@class='top-navigation__item-link'][@href='/careers']"));
+            action.MoveToElement(careersElement).Build().Perform();
+
+            var jobListingsElement = _chrome.FindElement
+            (By.XPath("//*[@class='top-navigation__main-link'][@href='/careers/job-listings']"));
+            action.MoveToElement(jobListingsElement).Build().Perform();
```

Comment:
Suggestion: there is no necessity to add another move :) you just can click on element :)